src/components/BlueButton.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <header class="bg-white">
+    <div class="max-w-7xl mx-auto py-2 sm:px-6 lg:px-8 text-black text-left">
+      <h1 :class="getClass()">
+        <slot />
+      </h1>
+    </div>
+  </header>
+</template>
+<script>
+export default {
+  name: 'Heading',
+  props: {
+    level: {
+      type: [Number, String],
```

Comment:
>>Wouldn't it make more sense to name the button semantically (eg. buttonCreate, buttonEdit, buttonDownload), rather than by colour, that way:
I can see the point and it definitely make sense. But, at present we haven't thought different styles for buttons based on their actions - but definitely something to think about in future for e.g if we are planning to have icons  (specifying these actions) displayed on buttons etc

>>Could you add a bit of documentation as to what level does. It seems to correspond to size?
Sure, will do

Reply:
> Afraid I have to go collect my daughter, so popping what I have in here.
> 
> I think my comments on BlueButton.vue might be because it isn't actually a button after-all, possibly its WIP?

Yes, it is. It is removed in latest PR. Sorry, that shouldn't have been in commit.